package.json

```diff
@@ -1,10 +1,54 @@
 {
+  "version": "0.4.0",
+  "build": {
+    "publish": [
+      "github"
+    ],
+    "appId": "TinyTacoTeam.ZazuApp",
+    "win": {
+      "target": [
+        "nsis"
+      ],
+      "icon": "resources/windows/icon.ico"
+    },
+    "nsis": {
+      "oneClick": true,
+      "installerHeaderIcon": "resources/windows/setup-icon.ico"
+    },
+    "linux": {
+      "target": [
+        "deb"
+      ]
+    },
+    "mac": {
+      "target": [
+        "dmg"
+      ],
+      "icon": "resources/osx/icon.icns",
+      "category": "public.app-category.productivity"
+    },
+    "dmg": {
+      "icon": "resources/osx/dmg-icon.icns",
+      "background": "resources/osx/dmg-background.png"
+    }
+  },
+  "directories": {
+    "buildResources": "resources"
+  },
```

Comment:
`directories` should be moved to `build` section, otherwise the following warning will be shown during the build:

```
⚠️  "directories" in the root is deprecated, please specify in the "build"
```